templates/node/src/client.ts.twig

```diff
@@ -271,6 +275,39 @@ class Client {
             data = await response.json();
         } else if (responseType === 'arrayBuffer') {
             data = await response.arrayBuffer();
+        } else if (response.headers.get('content-type')?.includes('multipart/form-data')) {
+            const body = await buffer(response.body);
+            const boundary = multipart.getBoundary(
+                response.headers.get("content-type") || ""
+            );
+            const parts = multipart.parse(body, boundary);
+            const partsObject: { [key: string]: any } = {};
+            
+            for (const part of parts) {
+                if (part.name) {
```

Comment:
is this nessessary? If so, lets avoid nesting. we can do opposite of that, and `continue`